src/lib.rs

```diff
@@ -23,7 +23,12 @@ mod windows;
 #[cfg(target_os = "windows")]
 use windows as platform;
 
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
+#[cfg(target_os = "linux")]
```

Comment:
```suggestion
#[cfg(any(target_os = "linux", target_os = "freebsd", target_os = "dragonfly", target_os = "netbsd", target_os = "openbsd"))]
```

also in Cargo.toml. I think `target_family = "unix"` includes macOS, iOS, and Android.

Reply:
Should we should rename the `linux.rs` file to `linux-bsd.rs`?

Reply:
Note that I haven't actually tested this on any BSDs, but I don't see any reason why this shouldn't work on BSDs.

Reply:
That has a nice ring to it, I will review all this when I get home. 👍🏼

Reply:
You renamed the file but didn't change the `cfg` macros yet.

Reply:
Oops!